apps/web-giddh/src/app/ocr-voucher/ocr-voucher-list/ocr-voucher-list.component.ts

```
@@ -0,0 +1,500 @@
+     * @param {*} [from]
+     * @return {*}  {void}
+     * @memberof ExportsComponent
+     */
+    public dateSelectedCallback(value?: any, from?: any): void {
+        if (value && value.event === "cancel") {
+            this.hideGiddhDatepicker();
+            return;
+        }
+        this.selectedRangeLabel = "";
+
+        if (value && value.name) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)